examples/cpp/Breakpoint.cc

```diff
@@ -0,0 +1,156 @@
+/*
+ * This is an example of a hardware breakpoint on a kernel address.
+ * run in project examples directory with:
+ * sudo ./breakpoint.py <0xaddress> <pid> <breakpoint_type>
```

Comment:
Maybe the length should be a parameter as well? Note that for HW_BREAKPOINT_X, the length is fixed as sizeof(long) per doc https://www.systutorials.com/docs/linux/man/2-perf_event_open/.